run.sh

```diff
@@ -0,0 +1,19 @@
+# note: must be run from the project root folder
```

Comment:
If we're going to add a script to test running things, let's add a write-up about it in the README.

Reply:
A section on this script has been added to the README.

Reply:
Let's expand that section a bit to include that we're expecting the jar to already exist or run a build in the script before executing the jar. That 2nd option may be the better one to ensure we have a fresh build before execution.

Reply:
Good call. The run.sh script has been altered to build the project before running it.